test/esmodules/fixtures/export-default.minified.js

```diff
@@ -0,0 +1 @@
+'use strict';export default function exported(a){console.log(a);console.log(1)};
```

Comment:
strict directive is excessive, modules are automatically in strict mode, could this be avoided?

Reply:
Good call. This can be removed post Closure transform similar to how exports are re-added.

Reply:
Addressed with 83011a3.